File: 프로그래머스/0/120907. OX퀴즈/OX퀴즈.py
def solution(quiz):
    result = []
    for q in quiz:
        div_q = q.split()
             
        if div_q[1] == '-':
            answer = int(div_q[0]) - int(div_q[2])
        elif div_q[1] == '+':
            answer = int(div_q[0]) + int(div_q[2])

        if answer == int(div_q[4]):
            result.append("O")
        else:
            result.append("X")
    return result

File: 프로그래머스/0/120907. OX퀴즈/OX퀴즈.js
function solution(quiz) {
    return quiz.map((q) => {
        const [X, operator, Y, equal, Z] = q.split(" ");
        
        let calc = 0;
        
        if(operator === "+"){
            calc = Number(X) + Number(Y);
        } else calc = Number(X) - Number(Y);
        
        return calc === Number(Z) ? "O" : "X";
    })
}